cms/static/js/views/pages/container.js

```diff
@@ -1138,7 +1143,6 @@ function($, _, Backbone, gettext, BasePage,
         onNewXBlock: function(xblockElement, scrollOffset, is_duplicate, data) {
             var useNewTextEditor = this.$('.xblock-header-primary').attr('use-new-editor-text'),
                 useNewVideoEditor = this.$('.xblock-header-primary').attr('use-new-editor-video'),
-                useVideoGalleryFlow = this.$('.xblock-header-primary').attr("use-video-gallery-flow"),
```

Comment:
Are we sure it's OK to remove this "video gallery flow"? Isn't it something that 2U uses?

Reply:
Oh no.. @DanielVZ96 had code in https://github.com/openedx/frontend-app-authoring/pull/1838#pullrequestreview-2793228930 which used this alternate video creation flow, but I didn't know it existed, and so I reverted those changes 😢 

But you're right, we should avoid breaking anything else, and fix what we removed.. sorry about this!

Reply:
@bradenmacdonald you are right. I'll have to refactor this a bit to handle both flows.

Reply:
@DanielVZ96 @pomegranited CC @bradenmacdonald I'm not sure if this is in scope, but when you open the editor from the Unity legacy page, with `new_core_editors.use_video_gallery_flow` enabled, the video gallery workflow doesn't open.

Reply:
@ChrisChV I think that is in scope. nice find.

Reply:
@ChrisChV I made this more backwards compatible. can you test it again please?

Reply:
I still see the issue: https://www.loom.com/share/e890c58a81e34a34a08c3fa708d1197d

Reply:
@DanielVZ96 I still can't see the gallery workflow when opening the editor from the legacy unit page. I have rebuilt the assets. Maybe my local is misconfigured, @pomegranited could you test if it works?

Reply:
@ChrisChV I didn't think we had to worry about the legacy unit page anymore -- Isn't that right @bradenmacdonald ?

Reply:
Not really, but we should at least avoid introducing regressions on it.

Reply:
Ok, make sense to me, I can confirm that is not a regression :+1: